vignettes/age-struct-pop.Rmd

```diff
@@ -78,7 +78,7 @@ By default `sim_linelist()` simulates individuals ages assuming a uniform distri
 
 ***Note***: all ages are assumed to be in the unit of years and need to be integers (or at least ["integerish"](https://rlang.r-lib.org/reference/is_integerish.html) if stored as double).
 
-All simulations in this vignette condition the simulation to have a minimum outbreak size (`min_outbreak_size`) of 100 cases to clearly visualise the distribution of ages.
+All simulations in this vignette condition the simulation to have a minimum outbreak size of 100 cases (`outbreak_size`, the maximum outbreak size is left at `1e5`) to clearly visualise the distribution of ages.
```

Comment:
I've been changing the mean of the contact dist and the probability of infection, and it's very helpful to be able to set a maximum number of cases so the process finishes much earlier and doesn't get stuck, as I mentioned above it'd be maybe better to have a smaller maximum outbreak size to help with this if users don't change the default values

Reply:
Resolved in d49c506d9bcb44929252114f3f3c2878e3ae3ba4.